eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/WorkOrderRest.java

```diff
@@ -31,7 +32,13 @@
 	@Consumes("application/json")
 	public Response readWorkOrder(@PathParam("workorder") String number) {
 		try {
-			return ok(inforClient.getWorkOrderService().readWorkOrder(authenticationTools.getInforContext(), number));
+			WorkOrder workOrder = inforClient.getWorkOrderService().readWorkOrder(authenticationTools.getInforContext(), number);
+
+			if ("IS".equals(workOrder.getTypeCode())) {
+				throw Tools.generateFault("Invalid work order");
+			}
```

Comment:
Why are IS type events generating a fault if the web service call does not fail?

Reply:
Another option would be to do this in the frontend, showing an error message when type === 'IS'.

Either way, 'IS' type work orders are generated when issuing equipments directly from a store to an equipment without using a work order. They are probably a work around and are not really "real" work orders, so it can be argued that it can make sense to generate a fault here.

Reply:
While that might be true, there might be use cases of people opening these work orders in EAM Light. If Infor EAM allows for it, we should too, unless there is a specific reason why we shouldn't.

Reply:
See EAM-1377. Maybe we should discuss this with Goran, or perhaps this feature should only be present when compiled with `-PCERN`?